Fakebook.Profile/Fakebook.Profile.RestApi/Controllers/ProfileController.cs

```diff
@@ -162,5 +162,46 @@ public async Task<ActionResult> UpdateAsync([FromBody] ProfileApiModel apiModel)
                 return BadRequest();
             }
         }
+
+        [HttpPost("follow/{user=user}")]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public async Task<ActionResult> Follow(ProfileApiModel user)
+        {
+            // Get emails of each user
+            string thisUserEmail = GetUserEmail();
+            string followEmail = user.Email;
+            if (thisUserEmail is null)
+            {
+                return NotFound(thisUserEmail);
```

Comment:
should be 401 unauthorized - though authorization filter probably should take care of that possibility, it's fine

Reply:
Changed to Unauthorized